tests/uncertainty/test_public_funcs.py

```diff
@@ -63,14 +63,15 @@ def test_get_warm_level_file_loading(mock_read_csv, mock_data_for_warm_level):
 
     # Set up the mock to return different DataFrames based on input file path
     def side_effect(file_path, **kwargs):
-        if file_path == gwl_1850_1900_file:
-            return mock_ipcc_df
-        elif file_path == gwl_1981_2010_file:
-            return mock_non_ipcc_df
-        elif file_path == "data/gwl_1981-2010ref_EC-Earth3_ssp370.csv":
-            return mock_ece3_df
-        else:
-            return pd.DataFrame()
+        match file_path:
+            case file_path if file_path == gwl_1850_1900_file:
```

Comment:
wildcard

Reply:
Changed in 2313c7b2a6bbbfee443755b838ce73a48bc90c8e